docs/0.4/examples/gameroom/walkthrough/index.md

```diff
@@ -1,4 +1,28 @@
-## I-1. Behind scene 1: Alice logs into Acme's Gameroom UI
+
+# Gameroom Technical Walkthrough
+
+***NOTE:*** This document is a work-in-progress! We are updating the
+walkthrough with details for Splinter 0.4 and converting it to markdown. For
```

Comment:
Not standard: Markdown is written with a capital M.

Reply:
fixed